lte/protos/mconfig/mconfigs.proto

```diff
@@ -381,3 +382,7 @@ message GatewayDNSConfigRecordsItems {
   repeated string cname_record = 3;
   string domain = 4;
 }
+
+message NHRoutes {
+    map<string, string> plmn_map = 1;
+}
```

Comment:
i created a message instead of using `map<string, string> plmn_map = 32` because we may need more than   `plmn_map`